grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatus.java

```diff
@@ -250,7 +251,12 @@ public static void reportStatus(HttpHeaders headers,
             status = addCause(status, grpcThrowable);
         }
 
-        final Metadata metadata = MetadataUtil.copyFromHeaders(headers);
+        final Metadata metadata = MetadataUtil.copyFromHeaders(
+                headers,
+                HttpHeaderNames.STATUS,
```

Comment:
Instead of passing these parameters, does it make sense to just have a constant with all four? I guess it makes sense to strip all of them in all cases.

Otherwise, let's not use varargs it's wasteful for these constants.